### PR TITLE
Fixes #47: document an example pytest fixture

### LINKED ISSUE #47: Provide a py.test fixture in addition to the decorator
When using py.test for tests it's usually nicer to activate features like this using a fixture. The usage with the fixture (assuming it's named `responses`) could be like this:

``` python
def test_my_api(responses):
    responses.add(responses.GET, 'http://twitter.com/api/1/foobar',
                  body='{"error": "not found"}', status=404,
                  content_type='application/json')

    # ...
```

I.e. instead of the decorator you'd use a `responses` fixture and instead of calling `add` as a global from the imported module (if you register it as a pytest plugin the user doesn't need to import it at all) you'd call it on the object returned by the fixture function (which could simple be the module `responses`, but a wrapper would be cleaner of course).

### PR DESCRIPTION
**Pull request by tomviner**
Fixes https://github.com/getsentry/responses/issues/47

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/responses/118)

### CODE REVIEW

**Comment by bmwant** [comment]
@tomviner Could you also add an example for usage of such a fixture? Add update indent for decorator, please, there is an extra space.

**Comment by tomviner** [comment]
Thanks for review @bmwant, I've added an example usage.

**Comment by latusaki** [comment]
This PR should be closed now?

File: README.rst [hunk 1]
```diff
@@ -147,6 +147,23 @@ Responses as a context manager
         resp = requests.get('http://twitter.com/api/1/foobar')
         resp.status_code == 404
 
+Responses as a pytest fixture
+-----------------------------
+
+.. code-block:: python
+
+    @pytest.yield_fixture
```
Comment (markstory) [inline README.rst]: Shouldn't this be `pytest.fixture`?

Comment (bmwant) [reply README.rst]: In newer versions - yes, but that's ok to have it in example, we do not pin to specific pytest version here

**Comment by markstory** [comment]
Rebased and merged in 3bc26d57bfbf02a570b6df30212a56f183349e06

**Pull request closed without merging**